From f8626cb1871a9729c12dc26a640f13863c93ab37 Mon Sep 17 00:00:00 2001
From: roberto crippa <roberto.crippa@intel.com>
Date: Wed, 14 May 2014 11:05:58 +0200
Subject: [PATCH 316/429] bluetooth: tether wlan network over PAN

we need a bridge in place to have bnep interface up.
So ethernet bridge has to be enabled in kernel.

Signed-off-by: roberto crippa <roberto.crippa@intel.com>
---
 arch/x86/configs/i386_edison_defconfig | 8 +++++++-
 1 file changed, 7 insertions(+), 1 deletion(-)

diff --git a/arch/x86/configs/i386_edison_defconfig b/arch/x86/configs/i386_edison_defconfig
index a9002b702f2c..0554980bf763 100644
--- a/arch/x86/configs/i386_edison_defconfig
+++ b/arch/x86/configs/i386_edison_defconfig
@@ -676,6 +676,7 @@ CONFIG_NETWORK_SECMARK=y
 CONFIG_NETFILTER=y
 # CONFIG_NETFILTER_DEBUG is not set
 CONFIG_NETFILTER_ADVANCED=y
+CONFIG_BRIDGE_NETFILTER=y
 
 #
 # Core Netfilter Configuration
@@ -781,6 +782,7 @@ CONFIG_NETFILTER_XT_MATCH_MULTIPORT=y
 # CONFIG_NETFILTER_XT_MATCH_OSF is not set
 # CONFIG_NETFILTER_XT_MATCH_OWNER is not set
 # CONFIG_NETFILTER_XT_MATCH_POLICY is not set
+# CONFIG_NETFILTER_XT_MATCH_PHYSDEV is not set
 # CONFIG_NETFILTER_XT_MATCH_PKTTYPE is not set
 CONFIG_NETFILTER_XT_MATCH_QUOTA=y
 # CONFIG_NETFILTER_XT_MATCH_RATEEST is not set
@@ -846,6 +848,7 @@ CONFIG_IP6_NF_TARGET_REJECT=y
 CONFIG_IP6_NF_MANGLE=y
 CONFIG_IP6_NF_RAW=y
 # CONFIG_IP6_NF_SECURITY is not set
+# CONFIG_BRIDGE_NF_EBTABLES is not set
 # CONFIG_IP_DCCP is not set
 # CONFIG_IP_SCTP is not set
 # CONFIG_RDS is not set
@@ -854,10 +857,13 @@ CONFIG_IP6_NF_RAW=y
 CONFIG_L2TP=y
 # CONFIG_L2TP_DEBUGFS is not set
 # CONFIG_L2TP_V3 is not set
-# CONFIG_BRIDGE is not set
+CONFIG_STP=y
+CONFIG_BRIDGE=y
+CONFIG_BRIDGE_IGMP_SNOOPING=y
 CONFIG_HAVE_NET_DSA=y
 # CONFIG_VLAN_8021Q is not set
 # CONFIG_DECNET is not set
+CONFIG_LLC=y
 # CONFIG_LLC2 is not set
 # CONFIG_IPX is not set
 # CONFIG_ATALK is not set
-- 
2.37.3

